src/app/room/components/room.component.html

```diff
@@ -1,3 +1,8 @@
+<div class="text-container" *ngIf="room$ | async as room">
+  <p *ngIf="room.typeOfGame === gameType.Winner">the last one will be the glorious winner 😀!</p>
+  <p *ngIf="room.typeOfGame === gameType.Loser">the last one will be the loser and never life out his shame 😀!</p>
```

Comment:
Ok don't let Marcos read this, but I think these days it should be `their` shame, so as to include all genders :)